src/actions/chatActions.js

```diff
@@ -0,0 +1,16 @@
+export const CHAT_ADD_MESSAGE = 'CHAT_ADD_MESSAGE';
+export const CHAT_SET_USER_STATUS = 'CHAT_SET_USER_STATUS';
+export const CHAT_CLEAR_MESSAGES = 'CHAT_CLEAR_MESSAGES';
+export const CHAT_SET_USERNAME = 'CHAT_SET_USERNAME';
+
+export const addMessage = ({ key, fromUsername, maybeSenderId, message }) =>
+    ({ type: CHAT_ADD_MESSAGE, key, fromUsername, maybeSenderId, message })
+
+export const clearMessages = () =>
+    ({ type: CHAT_CLEAR_MESSAGES })
+
+export const addUserStatus = ({ fromUsername, userStatus, created }) =>
+    ({ type: CHAT_SET_USER_STATUS, fromUsername, userStatus, created })
+
+export const setUsername = (username) =>
+    ({ type: CHAT_SET_USERNAME, username })
```

Comment:
@Spetastian There's extra parens here, yes?  And on these other functions, whose bodies can just be `{ ... }` rather than `({ ... })`, correct?

(And newline, please)